naff/ext/sentry.py

```diff
@@ -0,0 +1,79 @@
+"""
+Sets up a Sentry Logger
+
+And then call `bot.load_extension('naff.ext.sentry', token=SENTRY_TOKEN)`
+Optionally takes a filter function that will be called before sending the event to Sentry.
+"""
+import logging
+from typing import Any, Callable, Optional
+
+from naff.api.events.internal import Error
+
+try:
+    import sentry_sdk
+except ModuleNotFoundError:
+    logging.error("sentry-sdk not installed, cannot enable sentry integration")
```

Comment:
This should use the logger in `const` rather than its own separate thing, right?

Reply:
You are absolutely correct